modules/osdk-golang-tutorial-prereqs.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/golang/osdk-golang-tutorial.adoc
+
+[id="osdk-golang-tutorial-prereqs_{context}"]
```

Comment:
Unless you have a plan to reuse this information, I'd set the prereqs directly in the assembly.